smoke/IotEdgeQuickstart/details/IotedgedWindows.cs

```diff
@@ -132,7 +133,34 @@ public async Task Configure(string connectionString, string image, string hostna
             }
         }
 
-        public Task Start() => Task.CompletedTask; // Runtime starts automatically when it's installed
+        public async Task Start()
+        {
+            Console.WriteLine($"Current OS Version: {Environment.OSVersion}");
+
+            // Need to start up service explicitly as service is not started up automatically after configured in Windows 10 RS4
+            if (Environment.OSVersion.ToString().Equals("Microsoft Windows NT 6.2.9200.0"))
+            {
+                Console.WriteLine($"Starting up iotedged on Windows");
+
+                using (var cts = new CancellationTokenSource(TimeSpan.FromMinutes(2)))
+                {
+                    try
+                    {
+                        await Process.RunAsync("sc", "start iotedge", cts.Token);
+
+                        // Wait for service to become active
+                        while (ServiceController.GetServices().FirstOrDefault(s => s.ServiceName == "iotedge")?.Status != ServiceControllerStatus.Running)
```

Comment:
Don't we need to exit this loop if the two-minute cancellation token expires?

Reply:
thanks.